cumulusci/cli/tests/test_service.py

```diff
@@ -26,19 +28,18 @@ def test_service_list():
     }
     runtime.keychain._default_services = {"test": "test_alias", "bad": "bad_alias"}
 
-    result = run_cli_command("service", "list", runtime=runtime)
-    assert (
-        result.output
-        == """                            Services                             
-                                                                 
-  Default   Type             Name          Description           
- ─────────────────────────────────────────────────────────────── 
-  ✔         bad              bad_alias     Unconfigured Service  
-            something_else                 something else        
-            test             test2_alias   Test Service          
-  ✔         test             test_alias    Test Service          
-                                                                 
-"""  # noqa: W291,W293
+    run_cli_command("service", "list", runtime=runtime)
+
+    table_mock.assert_called_once_with(
+        [
+            ["Default", "Type", "Name", "Description"],
+            [True, "bad", "bad_alias", "Unconfigured Service"],
+            [False, "something_else", "", "something else"],
+            [False, "test", "test2_alias", "Test Service"],
+            [True, "test", "test_alias", "Test Service"],
+        ],
+        title="Services",
+        dim_rows=[2],
```

Comment:
I had intentionally changed it away from this approach because I couldn't tell by looking at the test whether it produces correct output.

I guess changing it back is motivated by the output not being the same on Windows? In which case this is probably the most pragmatic solution. But another option might be to run the output through a normalizer before making an assertion about it.